cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/Security.java

```diff
@@ -13,9 +13,18 @@
 
     private final List<String> cloudSecurityIds;
 
+    private final boolean useNetworkCidrAsSourceForDefaultRules;
+
     public Security(@Nonnull Collection<SecurityRule> rules, @Nonnull Collection<String> cloudSecurityIds) {
```

Comment:
Not a big deal but why not delegate to the other constructor below as `this(rules, cloudSecurityIds, false)`, so that the logic need not be duplicated?